### PR TITLE
UU: Fix skip link

### PR DESCRIPTION
**Pull request by PerOlavM**
- Fix skip-link (flyttet maincontent til page-template)
- Autocomplete på søkefelt
- Favicon
- Bumping og tsconfig

Har testet selv lokalt og i dev

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,24 +1,25 @@
 {
   "compilerOptions": {
-    "target": "ESNext",
+    "target": "es5",
```
Comment (anders-nom) [inline tsconfig.json:3]: Er det nødvendig å ha en så gammel target?

File: tsconfig.json [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
     "useDefineForClassFields": true,
     "lib": [
       "DOM",
       "DOM.Iterable",
       "ESNext"
     ],
-    "allowJs": false,
+    "allowJs": true,
```
Comment (anders-nom) [inline tsconfig.json:10]: Er denne nødvendig?

File: server/tsconfig.json [hunk 3]
```diff
@@ -19,8 +20,6 @@
     "resolveJsonModule": true
   },
   "include": [
-    "src/**/*.ts",
-    "../common/**/*.ts",
-    "global.d.ts"
+    "**/*.ts"
```
Comment (anders-nom) [inline server/tsconfig.json:23]: Med denne vil frontend-koden også bli med i scopet til server-koden, bør helst unngå dette.

**Comment by PerOlavM** [comment]
> Kudos, SonarCloud Quality Gate passed!    [Quality Gate passed](https://sonarcloud.io/dashboard?id=navikt_nav-office-search&pullRequest=48)
> 
> [Bug](https://sonarcloud.io/project/issues?id=navikt_nav-office-search&pullRequest=48&resolved=false&types=BUG) [A](https://sonarcloud.io/project/issues?id=navikt_nav-office-search&pullRequest=48&resolved=false&types=BUG) [0 Bugs](https://sonarcloud.io/project/issues?id=navikt_nav-office-search&pullRequest=48&resolved=false&types=BUG) [Vulnerability](https://sonarcloud.io/project/issues?id=navikt_nav-office-search&pullRequest=48&resolved=false&types=VULNERABILITY) [A](https://sonarcloud.io/project/issues?id=navikt_nav-office-search&pullRequest=48&resolved=false&types=VULNERABILITY) [0 Vulnerabilities](https://sonarcloud.io/project/issues?id=navikt_nav-office-search&pullRequest=48&resolved=false&types=VULNERABILITY) [Security Hotspot](https://sonarcloud.io/project/security_hotspots?id=navikt_nav-office-search&pullRequest=48&resolved=false&types=SECURITY_HOTSPOT) [A](https://sonarcloud.io/project/security_hotspots?id=navikt_nav-office-search&pullRequest=48&resolved=false&types=SECURITY_HOTSPOT) [0 Security Hotspots](https://sonarcloud.io/project/security_hotspots?id=navikt_nav-office-search&pullRequest=48&resolved=false&types=SECURITY_HOTSPOT) [Code Smell](https://sonarcloud.io/project/issues?id=navikt_nav-office-search&pullRequest=48&resolved=false&types=CODE_SMELL) [A](https://sonarcloud.io/project/issues?id=navikt_nav-office-search&pullRequest=48&resolved=false&types=CODE_SMELL) [0 Code Smells](https://sonarcloud.io/project/issues?id=navikt_nav-office-search&pullRequest=48&resolved=false&types=CODE_SMELL)
> 
> [No Coverage information](https://sonarcloud.io/component_measures?id=navikt_nav-office-search&pullRequest=48) No Coverage information [](https://sonarcloud.io/component_measures?id=navikt_nav-office-search&pullRequest=48&metric=new_duplicated_lines_density&view=list) [2.6% Duplication](https://sonarcloud.io/component_measures?id=navikt_nav-office-search&pullRequest=48&metric=new_duplicated_lines_density&view=list)

Gjelder testene. Dette er OK.

**Review by anders-nom — APPROVED** [review]

**Pull request merged**